tests/clv/models/test_gamma_gamma.py

```diff
@@ -179,26 +218,22 @@ def test_spend(self, distribution):
             )
 
     @pytest.mark.parametrize("distribution", (True, False))
-    def test_new_customer_spend(self, distribution):
+    def test_new_customer_spend(self, distribution, data):
         p_mean = 35
         q_mean = 15
         v_mean = 3
-
-        model = GammaGammaModel(
-            customer_id=self.z_mean_idx,
-            mean_transaction_value=self.z_mean,
-            frequency=self.z_mean_nobs,
+        custom_model_config = {
             # Narrow values
-            p_prior=pm.Normal.dist(p_mean, 0.01),
-            q_prior=pm.Normal.dist(q_mean, 0.01),
-            v_prior=pm.Normal.dist(v_mean, 0.01),
-        )
-
-        fake_fit = pm.sample_prior_predictive(
-            samples=1000, model=model.model, random_seed=self.rng
+            "p_prior": {"dist": "normal", "kwargs": {"mu": p_mean, "sigma": 0.01}},
+            "q_prior": {"dist": "normal", "kwargs": {"mu": q_mean, "sigma": 0.01}},
+            "v_prior": {"dist": "normal", "kwargs": {"mu": v_mean, "sigma": 0.01}},
+        }
+        model = GammaGammaModel(
+            data=data,
+            model_config=custom_model_config,
         )
-        fake_fit.add_groups(dict(posterior=fake_fit.prior))
-        model._fit_result = fake_fit
+        model.build_model()
+        model.fit(chains=1, progressbar=False, random_seed=self.rng)
```

Comment:
No need to call mcmc to get a fake fit, why was this changed?

Reply:
fit_result rewriting altered and now fit() could be removed for testing